pending/nvim.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+    BleachBit
+    Copyright (C) 2008-2019 Andrew Ziem
+    https://www.bleachbit.org
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+-->
+
+<cleaner id="nvim">
+  <label>Neovim</label>
+  <description>Editor</description>
+  <option id="history">
+    <label>History</label>
+    <description>Delete netrw directory history, command history</description>
+    <action search="file" command="delete" path="~/.config/nvim/.netrwhist"/>
+    <action search="file" command="delete" path="~/.local/share/nvim/shada/main.shada"/>
```

Comment:
The best practice is to use the XDG_ variables as described in the [documentation on variables](https://docs.bleachbit.org/cml/variables.html)

Reply:
Done, thanks for the advice.